mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientidentifiers/ClientIdentifiersFragment.java

```diff
@@ -178,12 +180,20 @@ public void onClickIdentifierOptions(final int position, View view) {
             public boolean onMenuItemClick(MenuItem item) {
                 switch (item.getItemId()) {
                     case R.id.menu_remove_identifier:
-                        mClientIdentifiersPresenter.deleteIdentifier(clientId,
-                                identifiers.get(position).getId(), position);
+                            new AlertDialog.Builder(getActivity(), R.style.MaterialAlertDialogStyle)
+                                .setMessage("Do you really want to delete?")
```

Comment:
Please don't use hardcoded strings inside the java code, first add the string in the `strings.xml` file then use dynamically.